ecl/data_source_ecl_network_fic_gateway_v2_test.go

```diff
@@ -0,0 +1,212 @@
+package ecl
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccNetworkV2FICGatewayDataSource_basic(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skip this test in short mode")
+	}
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheckFICGateway(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccNetworkV2FICGatewayDataSourceBasic,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ecl_network_fic_gateway_v2.fic_gateway_1", "id"),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "description", OS_FIC_GW_DESCRIPTION),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "fic_service_id", OS_FIC_SERVICE_ID),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "fic_gateway_id", OS_FIC_GW_ID),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "name", OS_FIC_GW_NAME),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "qos_option_id", OS_FIC_GW_QOS_OPTION_ID),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "status", OS_FIC_GW_STATUS),
+					resource.TestCheckResourceAttr(
+						"data.ecl_network_fic_gateway_v2.fic_gateway_1", "tenant_id", OS_TENANT_ID),
+				),
+			},
+		},
+	})
+}
+
+func TestAccNetworkV2FICGatewayDataSource_queries(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skip this test in short mode")
+	}
+
+	resource.Test(t, resource.TestCase{
```

Comment:
この関数内のテストケースも並列実行にしてください。
```suggestion
	resource.ParallelTest(t, resource.TestCase{
```